### ISSUE TITLE
Readable byte stream

### ISSUE THREAD

**Issue by jakearchibald** [WICG/compression#31]
Looking at the spec, it seems like these transforms create regular readable streams. Should they be readable **byte** streams?

**Comment by ricea** [comment]
Correct. This is essentially blocked behind https://github.com/whatwg/streams/issues/616. Which itself is blocked on any browser implementing byte streams, as well as the backlog of higher-priority work.

**Comment by jakearchibald** [comment]
> as well as the backlog of higher-priority work

hah, yeah, I know what you mean.
